project/1.18/assets/tinkers-construct/tconstruct/book/materials_and_you/zh_cn/furtherreading/fantastic_foundry_craft.json

```
@@ -3,6 +3,6 @@
   "recipe": "tconstruct:common/fantastic_foundry",
   "grid_size": "small",
   "description": [
-    { "text": "熔融金属中最热门的新技术。跳过冶炼炉，从熔铸炉开始！详情请参阅Namea所著§o奇异熔铸术§r。" }
+    { "text": "熔融金属中最热门的新技术。跳过冶炼炉，从熔铸炉开始！" }
```

```suggestion
    { "text": "熔融金属中最热门的新技术。跳过冶炼炉，从熔铸炉开始！详情请参阅Namea所著§o奇异熔铸术§r。" }
```
这里检查一下原文，如果我记得没错的话这里是我加的，和原文同步。

---

原文：`Hottest new technology in molten metals. Skip the smeltery, start with a foundry! Learn more in §oFantastic Foundry§r by Namea.`（那这个翻译官方那边得去改）